
Showing logs for build [32mmeta-seldonio-seldon-core-pr-19-2th74-16[0m stage [32mmeta-pipeline[0m and container [32mstep-credential-initializer-x6s6b[0m
{"level":"info","ts":1598525856.5289133,"logger":"fallback-logger","caller":"logging/config.go:69","msg":"Fetch GitHub commit ID from kodata failed: open /var/run/ko/refs/heads/0.8.0-ns-support: no such file or directory"}
{"level":"info","ts":1598525856.5294147,"logger":"fallback-logger","caller":"creds-init/main.go:40","msg":"Credentials initialized."}

Showing logs for build [32mmeta-seldonio-seldon-core-pr-19-2th74-16[0m stage [32mmeta-pipeline[0m and container [32mstep-working-dir-initializer-x84cl[0m
{"level":"info","ts":1598525857.6359005,"logger":"fallback-logger","caller":"logging/config.go:69","msg":"Fetch GitHub commit ID from kodata failed: open /var/run/ko/HEAD: no such file or directory"}
{"level":"info","ts":1598525857.6384346,"logger":"fallback-logger","caller":"bash/main.go:64","msg":"Successfully executed command \"sh -c mkdir -p /workspace/source\"; output "}

Showing logs for build [32mmeta-seldonio-seldon-core-pr-19-2th74-16[0m stage [32mmeta-pipeline[0m and container [32mstep-place-tools[0m

Showing logs for build [32mmeta-seldonio-seldon-core-pr-19-2th74-16[0m stage [32mmeta-pipeline[0m and container [32mstep-git-source-meta-seldonio-seldon-core-pr-19-2th74-ttnmz[0m
{"level":"info","ts":1598525865.1269,"logger":"fallback-logger","caller":"logging/config.go:69","msg":"Fetch GitHub commit ID from kodata failed: open /var/run/ko/refs/heads/0.8.0-ns-support: no such file or directory"}
{"level":"info","ts":1598525870.917042,"logger":"fallback-logger","caller":"git/git.go:103","msg":"Successfully cloned https://github.com/SeldonIO/seldon-core.git @ 4eb029e3d4a8d242058a6d6544494779f0b5716b in path /workspace/source"}

Showing logs for build [32mmeta-seldonio-seldon-core-pr-19-2th74-16[0m stage [32mmeta-pipeline[0m and container [32mstep-setup-builder-home[0m

Showing logs for build [32mmeta-seldonio-seldon-core-pr-19-2th74-16[0m stage [32mmeta-pipeline[0m and container [32mstep-git-merge[0m
Using SHAs from PULL_REFS=master:4eb029e3d4a8d242058a6d6544494779f0b5716b,1987:c6d9d94bab63249754c5846b8eca4c6a68d9ef5b
DEBUG: JX_LOG_LEVEL=error LC_ALL=C git fetch origin --unshallow c6d9d94bab63249754c5846b8eca4c6a68d9ef5b: 4eb029e3d4a8d242058a6d6544494779f0b5716b:
DEBUG: ran git fetch --unshallow origin c6d9d94bab63249754c5846b8eca4c6a68d9ef5b: 4eb029e3d4a8d242058a6d6544494779f0b5716b: in 
DEBUG: LC_ALL=C JX_LOG_LEVEL=error git branch
DEBUG: LC_ALL=C JX_LOG_LEVEL=error git checkout master
DEBUG: ran git checkout master in 
DEBUG: LC_ALL=C JX_LOG_LEVEL=error git reset --hard 4eb029e3d4a8d242058a6d6544494779f0b5716b
DEBUG: ran git reset --hard 4eb029e3d4a8d242058a6d6544494779f0b5716b in 
DEBUG: LC_ALL=C JX_LOG_LEVEL=error git clean -fd .
DEBUG: ran clean --force -d . in 
DEBUG: LC_ALL=C JX_LOG_LEVEL=error git merge c6d9d94bab63249754c5846b8eca4c6a68d9ef5b
error: error during merge: merging c6d9d94bab63249754c5846b8eca4c6a68d9ef5b into master: git output: Auto-merging operator/testing/machinelearning.seldon.io_seldondeployments.yaml
Auto-merging operator/controllers/seldondeployment_engine.go
Auto-merging operator/config/crd/bases/machinelearning.seldon.io_seldondeployments.yaml
Auto-merging operator/apis/machinelearning.seldon.io/v1/seldondeployment_types.go
Auto-merging executor/cmd/executor/main.go
CONFLICT (content): Merge conflict in executor/cmd/executor/main.go
Automatic merge failed; fix conflicts and then commit the result.: failed to run 'git merge c6d9d94bab63249754c5846b8eca4c6a68d9ef5b' command in directory '', output: 'Auto-merging operator/testing/machinelearning.seldon.io_seldondeployments.yaml
Auto-merging operator/controllers/seldondeployment_engine.go
Auto-merging operator/config/crd/bases/machinelearning.seldon.io_seldondeployments.yaml
Auto-merging operator/apis/machinelearning.seldon.io/v1/seldondeployment_types.go
Auto-merging executor/cmd/executor/main.go
CONFLICT (content): Merge conflict in executor/cmd/executor/main.go
Automatic merge failed; fix conflicts and then commit the result.'
[31m
Pipeline failed on stage 'meta-pipeline' : container 'step-git-merge'. The execution of the pipeline has stopped.[0m

Showing logs for build [32mmeta-seldonio-seldon-core-pr-19-2th74-16[0m stage [32mmeta-pipeline[0m and container [32mstep-credential-initializer-x6s6b[0m
{"level":"info","ts":1598525856.5289133,"logger":"fallback-logger","caller":"logging/config.go:69","msg":"Fetch GitHub commit ID from kodata failed: open /var/run/ko/refs/heads/0.8.0-ns-support: no such file or directory"}
{"level":"info","ts":1598525856.5294147,"logger":"fallback-logger","caller":"creds-init/main.go:40","msg":"Credentials initialized."}

Showing logs for build [32mmeta-seldonio-seldon-core-pr-19-2th74-16[0m stage [32mmeta-pipeline[0m and container [32mstep-working-dir-initializer-x84cl[0m
{"level":"info","ts":1598525857.6359005,"logger":"fallback-logger","caller":"logging/config.go:69","msg":"Fetch GitHub commit ID from kodata failed: open /var/run/ko/HEAD: no such file or directory"}
{"level":"info","ts":1598525857.6384346,"logger":"fallback-logger","caller":"bash/main.go:64","msg":"Successfully executed command \"sh -c mkdir -p /workspace/source\"; output "}

Showing logs for build [32mmeta-seldonio-seldon-core-pr-19-2th74-16[0m stage [32mmeta-pipeline[0m and container [32mstep-place-tools[0m

Showing logs for build [32mmeta-seldonio-seldon-core-pr-19-2th74-16[0m stage [32mmeta-pipeline[0m and container [32mstep-git-source-meta-seldonio-seldon-core-pr-19-2th74-ttnmz[0m
{"level":"info","ts":1598525865.1269,"logger":"fallback-logger","caller":"logging/config.go:69","msg":"Fetch GitHub commit ID from kodata failed: open /var/run/ko/refs/heads/0.8.0-ns-support: no such file or directory"}
{"level":"info","ts":1598525870.917042,"logger":"fallback-logger","caller":"git/git.go:103","msg":"Successfully cloned https://github.com/SeldonIO/seldon-core.git @ 4eb029e3d4a8d242058a6d6544494779f0b5716b in path /workspace/source"}

Showing logs for build [32mmeta-seldonio-seldon-core-pr-19-2th74-16[0m stage [32mmeta-pipeline[0m and container [32mstep-setup-builder-home[0m

Showing logs for build [32mmeta-seldonio-seldon-core-pr-19-2th74-16[0m stage [32mmeta-pipeline[0m and container [32mstep-git-merge[0m
Using SHAs from PULL_REFS=master:4eb029e3d4a8d242058a6d6544494779f0b5716b,1987:c6d9d94bab63249754c5846b8eca4c6a68d9ef5b
DEBUG: JX_LOG_LEVEL=error LC_ALL=C git fetch origin --unshallow c6d9d94bab63249754c5846b8eca4c6a68d9ef5b: 4eb029e3d4a8d242058a6d6544494779f0b5716b:
DEBUG: ran git fetch --unshallow origin c6d9d94bab63249754c5846b8eca4c6a68d9ef5b: 4eb029e3d4a8d242058a6d6544494779f0b5716b: in 
DEBUG: LC_ALL=C JX_LOG_LEVEL=error git branch
DEBUG: LC_ALL=C JX_LOG_LEVEL=error git checkout master
DEBUG: ran git checkout master in 
DEBUG: LC_ALL=C JX_LOG_LEVEL=error git reset --hard 4eb029e3d4a8d242058a6d6544494779f0b5716b
DEBUG: ran git reset --hard 4eb029e3d4a8d242058a6d6544494779f0b5716b in 
DEBUG: LC_ALL=C JX_LOG_LEVEL=error git clean -fd .
DEBUG: ran clean --force -d . in 
DEBUG: LC_ALL=C JX_LOG_LEVEL=error git merge c6d9d94bab63249754c5846b8eca4c6a68d9ef5b
error: error during merge: merging c6d9d94bab63249754c5846b8eca4c6a68d9ef5b into master: git output: Auto-merging operator/testing/machinelearning.seldon.io_seldondeployments.yaml
Auto-merging operator/controllers/seldondeployment_engine.go
Auto-merging operator/config/crd/bases/machinelearning.seldon.io_seldondeployments.yaml
Auto-merging operator/apis/machinelearning.seldon.io/v1/seldondeployment_types.go
Auto-merging executor/cmd/executor/main.go
CONFLICT (content): Merge conflict in executor/cmd/executor/main.go
Automatic merge failed; fix conflicts and then commit the result.: failed to run 'git merge c6d9d94bab63249754c5846b8eca4c6a68d9ef5b' command in directory '', output: 'Auto-merging operator/testing/machinelearning.seldon.io_seldondeployments.yaml
Auto-merging operator/controllers/seldondeployment_engine.go
Auto-merging operator/config/crd/bases/machinelearning.seldon.io_seldondeployments.yaml
Auto-merging operator/apis/machinelearning.seldon.io/v1/seldondeployment_types.go
Auto-merging executor/cmd/executor/main.go
CONFLICT (content): Merge conflict in executor/cmd/executor/main.go
Automatic merge failed; fix conflicts and then commit the result.'
[31m
Pipeline failed on stage 'meta-pipeline' : container 'step-git-merge'. The execution of the pipeline has stopped.[0m
